tests/unit/browser/test_webelem.py

```diff
@@ -325,6 +327,38 @@ def test_debug_text(self, elem, xml, expected):
         elem._elem.toOuterXml.return_value = xml
         assert elem.debug_text() == expected
 
+    def test_remove_blank_target(self):
+        elem = get_webelem(tagname='a', attributes={'target': '_blank'})
+        elem.remove_blank_target()
+        assert elem._elem.attribute('target') == '_top'
+
+        for target in ['_self', '_parent', '_top', '']:
+            elem = get_webelem(tagname='a', attributes={'target': target})
+            elem.remove_blank_target()
+            assert elem._elem.attribute('target') == target
```

Comment:
As an example of a test rewritten inside a class, I'd rewrite this like this to use [`@pytest.mark.parametrize`](http://pytest.org/latest/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions) to parametrize the target and also test with `area`:

```python
class TestRemoveBlankTarget:

    # ...

    @pytest.mark.parametrize('tagname', ['a', 'area'])
    @pytest.mark.parametrize('target', ['_self', '_parent', '_top', ''])
    def test_keeping_target(tagname, target):
        elem = get_webelem(tagname=tagname, attributes={'target': target})
        elem.remove_blank_target()
        assert elem._elem.attribute('target') == target
```